.gitignore

```diff
@@ -11,26 +13,30 @@
 *.gbc
 *.gb
 
+# save game files
+*.sgm
+*.sav
+*.rtc
+*.sn*
+*.sa*
+*.sg1
+
 # rgbds extras
 *.map
 *.sym
 
-# converted image data
-*.2bpp
-*.1bpp
-*.pic
+# vim configuration
+# http://www.vim.org/scripts/script.php?script_id=441
+.lvimrc
 
-# swap files for vim
+# swap files for vim and gedit
 .*.swp
+*~
 
-# no data from extras/
-*.json
-
-# for any of the poor souls with save game files in their working directory
-*.sgm
-*.sav
-*.sys
+# macos files
+.DS_STORE
```

Comment:
(I might be in the minority but) I'm not a fan of keeping user system-specific files listed in the `.gitignore`.

Reply:
That's again just to stay in sync with gen 2. I'll remove it if you want.

Reply:
That would be my preference.